File: openapi/code_samples/java/api_v1_organizations_{org_id}_roles/post.java
CreateOrganizationRoleResponse res = scalekitClient.roles().createOrganizationRole(
    "org_123",
    CreateOrganizationRoleRequest.newBuilder()
        .setOrgId("org_123")
        .setRole(
            CreateOrganizationRole.newBuilder()
                .setName("org_admin")
                .setDisplayName("Org Admin")
                .setDescription("Organization-scoped role")
                .setExtends("base_role_name")          // optional
                .addPermissions("perm.read")           // optional
                .addPermissions("perm.write")          // optional
                .build()
        )
        .build()
);

File: openapi/code_samples/go/api_v1_organizations_{org_id}_roles/post.go
resp, err := scalekitClient.Role().CreateOrganizationRole(ctx, "org_123", &rolesv1.CreateOrganizationRole{

	Name:        "org_admin",

	DisplayName: "Org Admin",

	Description: proto.String("Organization-scoped role"), // optional

	Extends:     proto.String("base_role_name"),        // optional

	Permissions: []string{"perm.read", "perm.write"},   // optional

})